src/platform/Kconfig

```diff
@@ -5,7 +5,7 @@ menu "Platform"
 choice
```

Comment:
Would be good to add a mention that these platforms will be supported in the stable-v2.2 branch.